packages/fiori3/src/webComponents/Card/index.tsx

```diff
@@ -6,6 +6,7 @@ export interface CardPropTypes extends WithWebComponentPropTypes {
   heading?: string; // @generated
   subtitle?: string; // @generated
   status?: string; // @generated
+  headerInteractive?: boolean; // @generated
```

Comment:
We could add headerInteractive property to Card Demo. But could also be separate PR because Card demo looks kind of messy in general.

Reply:
is fixed